CONTRIBUTING.md

```diff
@@ -154,25 +166,23 @@ npm run start-server-dev
 
 It binds to `0.0.0.0:3001` by default.
 
-It is mainly a socket.io app + express.js.
-
-express.js is used for:
+It is mainly a `socket.io`-app, but includes `express.js` to serve:
```

Comment:
I think a more accurate description would be

The backend is an `express.js` server with `socket.io` integrated. It uses `socket.io` to communicate with clients, and most server logic is encapsulated in the `socket.io` handlers. `express.js` is also used to serve:

Reply:
```suggestion
The backend is an `express.js` server with `socket.io` integrated.
It uses `socket.io` to communicate with clients, and most server logic is encapsulated in the `socket.io` handlers.
`express.js` is also used to serve:
```